x-pack/plugins/apm/public/components/app/mobile/service_overview/metrics_map/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { Filter } from '@kbn/es-query';
+import { EuiSuperSelect, EuiText } from '@elastic/eui';
+
+import { EmbeddedMap } from './embedded_map';
+
+export enum MapTypes {
+  HTTP = 'http_requests',
+  SESSIONS = 'unique_sessions',
+}
+
+export function MetricsMap({
+  start,
+  end,
+  kuery,
+  filters,
+  comparisonEnabled,
+}: {
+  start: string;
+  end: string;
+  kuery?: string;
+  filters: Filter[];
+  comparisonEnabled: boolean;
+}) {
+  const [selectedMap, setMap] = useState(MapTypes.HTTP);
+
+  const availableMaps = [
+    {
+      id: MapTypes.HTTP,
+      label: i18n.translate(
+        'xpack.apm.serviceOverview.embeddedMap.dropdown.http.requests',
+        {
+          defaultMessage: 'HTTP requests',
+        }
+      ),
+      description: i18n.translate(
+        'xpack.apm.serviceOverview.embeddedMap.dropdown.http.requests.subtitle',
+        {
+          defaultMessage:
+            'HTTP defines a set of request methods to indicate the desired action to be performed for a given resource',
+        }
+      ),
+    },
+    {
+      id: MapTypes.SESSIONS,
+      label: i18n.translate(
+        'xpack.apm.serviceOverview.embeddedMap.dropdown.sessions',
+        {
+          defaultMessage: 'Sessions',
+        }
+      ),
+      description: i18n.translate(
+        'xpack.apm.serviceOverview.embeddedMap.dropdown.sessions.subtitle',
+        {
+          defaultMessage:
+            'An application session begins when a user starts an application and ends when the application exits.',
+        }
+      ),
+    },
+  ];
```

Comment:
This looks like a static array, can you move it outside the component so it doesn't trigger unnecessary renders on its children?

Reply:
good point! moved the array  https://github.com/elastic/kibana/commit/e7dd9513cfb75b98918c226c351ae395c31f1de3#diff-c545239b280d0c02ba7ec9dd9c565e6079126b85c6a5674e53c9d3a87e1856c5R17